### PR TITLE
Added section "Installing Magento Extensions"

### PR DESCRIPTION
**Pull request by mcclede**
**Description of the change**

Extends the README.md with the steps necessary to add an extension to the Bitnami Magento image.

**Benefits**

Adds information not currently in the document improving usability of the image especially for novice users.

**Possible drawbacks**

None

**Applicable issues**

#212 

**Additional information**

None

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -115,6 +115,39 @@ $ docker run -d --name magento \
 
 Access your application at *http://your-ip/*
 
+## Installing Magento Extensions
+
+There are a large number of Magento extensions used to add features to your Magento Stores. If you want to install an extension to your Magento container, these are the basic steps you need to take:
+
+#### Step 1: Log into the container shall as root
```
Comment (migruiz4) [inline README.md:122]: There is a typo in this line:

```suggestion
#### Step 1: Log into the container shell as root
```

File: README.md [hunk 2]
```diff
@@ -115,6 +115,39 @@ $ docker run -d --name magento \
 
 Access your application at *http://your-ip/*
 
+## Installing Magento Extensions
```
Comment (migruiz4) [inline README.md:118]:
```suggestion
## Installing Magento extensions
```

File: README.md [hunk 3]
```diff
@@ -115,6 +115,39 @@ $ docker run -d --name magento \
...
+
+#### Step 4: Follow the installation instructions for the extension. The Magento standard is to use composer.
+
+```console
+$ composer require <exntension name>
+$ php bin/magento module:enable <extension name>
+$ php bin/magento setup:upgrade
+$ php bin/magento setup:di:compile
+$ php bin/magento setup:static-content:deploy -f
+$ php bin/magento cache:flush```
```
Comment (migruiz4) [inline README.md:148]: Fix typo and additional (```)
```suggestion
$ composer require <extension name>
$ php bin/magento module:enable <extension name>
$ php bin/magento setup:upgrade
$ php bin/magento setup:di:compile
$ php bin/magento setup:static-content:deploy -f
$ php bin/magento cache:flush
```

**Review by migruiz4 — CHANGES_REQUESTED** [review]
Hi @mcclede,

Thank you very much for your contribution! I have requested several changes, could you please have a look?

File: README.md (hunk 1 above)
Comment (mcclede) [reply README.md]: Agree, fixed.

File: README.md (hunk 2 above)
Comment (mcclede) [reply README.md]: Agree, fixed.

File: README.md (hunk 3 above)
Comment (mcclede) [reply README.md]: Agree, fixed

**Pull request merged**